### DISCUSSION TITLE
Verification error

### DISCUSSION THREAD

**Discussion by aleeeeXcoo1** [blockscout/blockscout#8414] [Support]
I want to verify my smart contract `0xd2059de903dA9e01463d73FE702E8fbf58Fbdd6C `with the flattened file but get an error "No contract could be verified with provided data".
I don't understand and can't find the solution because names, licenses, and all other things are correct. What's wrong?

**Comment by ulyanas** [comment]
@aleeeeXcoo1 Hi! Could you please provide that contract's code, params and the chain id, please?

**Comment by alfarkas** [comment]
I have the same problem and it only happens with smart contracts deployed with a proxy. When validating the contract behind the proxy it works well, but not when attempting to validate the proxy. Did you find out a solution?

**Reply by ulyanas** [reply]
@alfarkas  Hi! Could you please point out to the problematic contract?

**Reply by alfarkas** [reply]
Hi, thank you for answering. I have experimented this with multiple contracts, always with proxys, in this particular case is [this one](https://testexplorer.lachain.network/address/0x3B46F12384f875D393875BDf3b7eb70319dd22c8/contracts#address-tabs) the code can be found [here](https://github.com/ripio/amigos-sol/blob/main/contracts/Amigos.sol).
